src/js/components/charts/TimeSeriesChart.js

```diff
@@ -457,9 +458,9 @@ var TimeSeriesChart = React.createClass({
         <svg height={props.height} width={props.width}>
           <g transform={"translate(" + margin.left + "," + margin.top + ")"}>
             <ChartStripes count={4} height={height} width={width} />
-            <g className="bars grid-graph" ref="grid" />
-            <g className="y axis" ref="yAxis" />
-            <g className="x axis" ref="xAxis" />
+            <g className="bars grid-graph" ref={ref => (this.gridRef = ref)} />
```

Comment:
on dcos-ui we can consider going with `React.createRef` it's reactjs-components that should be compatible. There should have been a branch by @brandonc somewhere

Reply:
@nLight This is based of the branch of @brandonc, if we are talking about the same one

Reply:
Ok, I understand the situation better now: This was done because `createReactClass` does not have the ability to add a constructor.